src/library/f3d_options.h

```diff
@@ -40,11 +41,17 @@ class F3D_EXPORT options
   std::vector<int> getAsIntVector(const std::string& name) const;
   std::vector<double> getAsDoubleVector(const std::string& name) const;
 
+  bool& getAsBoolRef(const std::string& name);
+  int& getAsIntRef(const std::string& name);
+  double& getAsDoubleRef(const std::string& name);
+  std::string& getAsStringRef(const std::string& name);
+  std::vector<int>& getAsIntVectorRef(const std::string& name);
+  std::vector<double>& getAsDoubleVectorRef(const std::string& name);
+
 private:
   class F3DInternals;
   F3DInternals* Internals;
   options(const options& opt) = delete;
```

Comment:
I don't think we should delete the copy constructor if we implement the copy assignment.

Reply:
ok, i'll add it back